index.html

```diff
@@ -995,42 +995,104 @@ <h1 class="component-title">Input</h1>
             </p>
 
             <section class="container margin-t-b-2r">
-              <input
-                type="text "
-                class="textbox margin-b-1r"
-                placeholder="Something"
-              />
-              <input
-                type="text"
-                class="textbox textbox-success margin-b-1r"
-                placeholder="Success"
-              />
-              <input
-                type="text"
-                class="margin-b-1r textbox textbox-fail"
-                placeholder="fail"
-              />
+              <form action="" class="document-form">
+                <div class="document-form-inputs">
+                  <label for="textbox-default" class="label"
+                    >Default Label</label
+                  >
+                  <input
+                    id="textbox-default"
+                    type="text "
+                    class="textbox margin-b-1r"
+                    placeholder="Something"
+                  />
+                </div>
+                <div class="document-form-inputs">
+                  <label class="label label-success" for="textbox-success"
+                    >Success Label</label
```

Comment:
You've capitalised "Success Label" but not "fail label" or "disabled label". your code looks clean. good to merge.

Reply:
also, your yellow snackbar's close button on hover looses contrast or visibility. you can change that in a later commit :).